include/dxguids/dxguids.h

```diff
@@ -36,8 +36,7 @@ __CRT_UUID_DECL(InterfaceName, l, w1, w2, b1, b2, b3, b4, b5, b6, b7, b8) \
 template <> constexpr GUID uuidof<InterfaceName>() \
 { \
     return { l, w1, w2, { b1, b2,  b3,  b4,  b5,  b6,  b7,  b8 } }; \
-} \
-static_assert(ConstexprIsEqualGUID(uuidof<InterfaceName>(), __uuidof(InterfaceName)), "GUID definition mismatch: "#InterfaceName);
```

Comment:
Should we maybe guard this in a `#if __cplusplus >= 201703L` block instead of deleting it?

Reply:
fine with that:)